### DISCUSSION TITLE
Support Conditional Create flow to automatically opt in users to passkeys.

### DISCUSSION THREAD

**Discussion by nicksrandall** [MasterKale/SimpleWebAuthn#581] [Ideas]
More info: https://developer.apple.com/videos/play/wwdc2024/10125/

When app detects that browser supports conditional create and user has been verified using an alternative method, the app can automatically create a passkey for that user and have it stored in their password manager and linked to their account. 

It would be great if this library supported the “mediation”: “conditional” option for creating a passkey.

**Comment by MasterKale** [comment]
I definitely intend to support this in SimpleWebAuthn! I've been playing around with Safari 18.0 in Dev Beta 1 to figure out what Safari's "internal conditions" means in Apple's flow chart:

![Screenshot 2024-06-11 at 1 17 58 PM](https://github.com/MasterKale/SimpleWebAuthn/assets/5166470/6a4e6167-6f1b-43d0-9730-aaa248da49fc)

Stay tuned 👀

**Reply by MasterKale** [reply]
I've created an issue to help me keep track of adding this to SimpleWebAuthn https://github.com/MasterKale/SimpleWebAuthn/issues/582
